packages/front-end/services/secret-api-keys.utils.ts

```diff
@@ -0,0 +1,38 @@
+import { ApiKeyInterface } from "back-end/types/apikey";
+
+type GroupedApiKeys = {
+  secret: ApiKeyInterface[];
+  user: ApiKeyInterface[];
+  readonly: ApiKeyInterface[];
+};
+
+/**
+ * The GET /keys endpoint returns all kinds of API keys.
+ * For the new API keys, we only care about the following criteria:
+ *  - Only include secret keys
+ *  - Secret keys have no type
+ *  - User keys have type 'user'
+ *  - Read-only keys have type 'read-only'
+ * @param apiKeys List of {@link ApiKeyInterface}
+ */
+export function groupApiKeysByType(apiKeys: ApiKeyInterface[]): GroupedApiKeys {
```

Comment:
Is this used any more?

Reply:
It is. If you find on the PR/files page you should see it [here](https://github.com/growthbook/growthbook/pull/1265/files#diff-e53ac8e3bfd54803b3be3e2acd31a9a0e44f45333f04620cc1c2183085816cceR79).